docs/high/file.rst

```diff
@@ -115,6 +115,13 @@ of supported drivers and their options:
         The argument values must be ``bytes`` objects. All three arguments are
         required to activate AWS authentication.
 
+        .. note::
+           Pre-built h5py packages on PyPI do not include this S3 support. If
```

Comment:
```suggestion

           Pre-built h5py packages on PyPI do not include this S3 support. If
```

Doesn't rst require a blank line here?

Reply:
nvm, https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html in pretty clear I am wrong.